tools/testbuild.sh

```diff
@@ -198,8 +198,8 @@ function build {
 function dotest {
   echo "===================================================================================="
   config=`echo $1 | cut -d',' -f1`
-  re=\\b${config/\//:}\\b
-  if [[ $blacklist =~ $re ]]; then
+  re="-${config/\//:}[[:space:]]"
+  if [[ "${blacklist} " =~ $re ]]; then
     echo "Skipping: $1"
```

Comment:
Isn't the test redundant anyway?
`dotest` is called with configs from `testlist` which is the opposite of `blacklist`.

> I couldn't find where "\b" was documented.

How about this one: https://www.rexegg.com/regex-boundaries.html#wordboundary

Reply:
testlist can contain eg. "/sim" which means all configs under boards/sim.
in that case, blacklist processing here is necessary.
it's used by the CI.

btw, thank you for the link.

Reply:
You're right, thanks.
if we are only interested in the substring, can we just use grep?
```bash
  config=`echo $1 | cut -d',' -f1 | sed -e "s/\//:/g"`
  if grep -q "$config" <<< "$blacklist" ; then
  # or
  # if [[ "$blacklist" == *"$config"* ]]; then
    echo "Skipping: $1"
```
Which one do you think is more readable? I'm ready to merge your changes as is if you prefer them.

Reply:
i guess a simple grep doesn't work well as a config can be a substring of other configs.
eg. sim:nx and sim:nxffs.